configuration/multimedia.md

```diff
@@ -30,128 +30,146 @@ In the last step we will create an AppVM from this template.
 
 `qvm-clone debian-9 t-multimedia`
 
-As we need to install some package outside of the regular repositories you need to enable networking for the Template VM.
-
-`qvm-prefs --set t-multimedia netvm sys-firewall`
-
 Launch a Terminal in the new template VM:
 
 `qvm-run --auto t-multimedia gnome-terminal`
 
 Important:
 Enter all the following commands in the terminal of the template VM
-
-Become the root user to run all follwoing command without the need to use sudo
+Become the root user to run all following command without the need to use sudo in the multimedia template VM
 
 `sudo -i`
 
+
 Installation of Spotify
 -----------------------
 
 Import GPG-Key for spotify
+As the template VM can't connect to internet you need to get the public key file from another AppVM and copy it to the template VM. The easiest way is to use the Qubes Clipboard to copy the keys from the AppVM where you get the key to the Template VM.
 
-`apt-key adv --keyserver hkp://keyserver.ubuntu.com:80 --recv-keys 0DF731E45CE24F27EEEB1450EFDC8610341D9410`
+In an AppVM which has Internet access:
+- Open http://keyserver.ubuntu.com:11371/pks/lookup?op=get&search=0xEFDC8610341D9410
+- Copy content of page to the Qubes Clipboard (Ctrl+C and then Shift+Ctrl+C)
 
-> http://keyserver.ubuntu.com:11371/pks/lookup?op=vindex&search=0xEFDC8610341D9410&fingerprint=on
-> UUID: Spotify Public Repository Signing Key <tux@spotify.com>
-> Key-ID: 0x341D9410
-> Fingerprint=0DF7 31E4 5CE2 4F27 EEEB  1450 EFDC 8610 341D 9410 
+Switch to the gnome terminal in the Multimedia Template VM
 
-Add Spotify repository to package list
+`nano spotify.pubkey`
 
-`echo deb http://repository.spotify.com stable non-free | tee /etc/apt/sources.list.d/spotify.list`
+Paste the content from the Qubes Clipboard into nano (Shift+Ctrl+V and then Paste)
+Save the file (Ctrl+O <Enter> Ctrl+X)
 
-Update package list
+Add the public key to the repository keyring
+`apt-key add spotify.pubkey`
 
-`apt-get update`
+Verify Fingerprint with
+
+`apt-key finger spotify`
+
+You can (and should) lookup the fingerprint on the keyserver:
+http://keyserver.ubuntu.com:11371/pks/lookup?op=vindex&search=0xEFDC8610341D9410&fingerprint=on
```

Comment:
Verification should be against some independent source. You can simply put the fingerprint here.
Also, it's a bit late for verification - you've already told apt to trust this key. Better call `gpg --with-fingerprint` on the file first, and `apt-key add` only if fingerprint matches.